### PR TITLE
Feature/cloud hub app suffix

### PR DESCRIPTION
**Pull request by TimGonz-AVIO**

### CODE REVIEW

File: library/src/main/java/com/avioconsulting/mule/deployment/api/models/CloudhubDeploymentRequest.groovy [hunk 1]
```diff
@@ -103,12 +109,26 @@ class CloudhubDeploymentRequest extends FileBasedAppDeploymentRequest {
...
         this.cloudHubAppPrefix = cloudHubAppPrefix
+        this.cloudHubAppSuffix = cloudHubAppSuffix
         this.appProperties = appProperties
         this.otherCloudHubProperties = otherCloudHubProperties
         if (this.appName.contains(' ')) {
             throw new Exception("Runtime Manager does not like spaces in app names and you specified '${this.appName}'!")
         }
-        def newAppName = "${cloudHubAppPrefix}-${this.appName}-${environment}"
+        def newAppName
+        if (cloudHubAppSuffix != null) {
```
Comment (adesjardin) [inline library/src/main/java/com/avioconsulting/mule/deployment/api/models/CloudhubDeploymentRequest.groovy:119]: Why are we supporting an arbitrary value for suffix, do we actually have a use case for that?  Do we need to have that option or could it just be append environment or don't have suffix?    

If we do want to have that option setting suffix to empty string is a little bit of an ambiguous why to disable the environment suffix.

Comment (TimGonz-AVIO) [reply library/src/main/java/com/avioconsulting/mule/deployment/api/models/CloudhubDeploymentRequest.groovy:119]: One of the usecases was for site specific deployments of applications to the same CH environment

Comment (TimGonz-AVIO) [reply library/src/main/java/com/avioconsulting/mule/deployment/api/models/CloudhubDeploymentRequest.groovy:119]: Im not aware of other usecases besides environment however

Comment (adesjardin) [reply library/src/main/java/com/avioconsulting/mule/deployment/api/models/CloudhubDeploymentRequest.groovy:119]: Let's chat through this tomorrow before we merge this in

**Pull request closed without merging**